src/main/java/org/jabref/logic/citationkeypattern/CitationKeyPattern.java

```diff
@@ -1,5 +1,93 @@
 package org.jabref.logic.citationkeypattern;
 
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public record CitationKeyPattern(String stringRepresentation) {
     public static final CitationKeyPattern NULL_CITATION_KEY_PATTERN = new CitationKeyPattern("");
+
+    // region - Author-related field markers
+    public static final CitationKeyPattern AUTHOR_YEAR = new CitationKeyPattern("[auth]");
+    public static final CitationKeyPattern AUTHOR_FIRST_FULL = new CitationKeyPattern("[authFirstFull]");
+    public static final CitationKeyPattern AUTHOR_FORE_INI = new CitationKeyPattern("[authForeIni]");
+    public static final CitationKeyPattern AUTHOR_ETAL = new CitationKeyPattern("[auth.etal]");
+    public static final CitationKeyPattern AUTHOR_ET_AL = new CitationKeyPattern("[authEtAl]");
+    public static final CitationKeyPattern AUTHOR_AUTH_EA = new CitationKeyPattern("[auth.auth.ea]");
+    public static final CitationKeyPattern AUTHORS = new CitationKeyPattern("[authors]");
+    public static final CitationKeyPattern AUTHORS_N = new CitationKeyPattern("[authorsN]");
+    public static final CitationKeyPattern AUTH_INI_N = new CitationKeyPattern("[authIniN]");
+    public static final CitationKeyPattern AUTH_N = new CitationKeyPattern("[authN]");
+    public static final CitationKeyPattern AUTH_N_M = new CitationKeyPattern("[authN_M]");
+    public static final CitationKeyPattern AUTHOR_INI = new CitationKeyPattern("[authorIni]");
+    public static final CitationKeyPattern AUTH_SHORT = new CitationKeyPattern("[authshort]");
+    public static final CitationKeyPattern AUTHORS_ALPHA = new CitationKeyPattern("[authorsAlpha]");
+    public static final CitationKeyPattern AUTHORS_ALPHA_LNI = new CitationKeyPattern("[authorsAlphaLni]");
```

Comment:
```suggestion
    public static final CitationKeyPattern AUTHORS_ALPHA_LNI = new CitationKeyPattern("[authorsAlphaLNI]");
```

Double check the others once too.